### ISSUE TITLE
[Bug]: Electron app crash on clicking a link whose target is _blank in webview when nativeWindowOpen: true and allowpopups enabled on MacOS

### ISSUE THREAD

**Issue by alokmahor** [electron/electron#31127]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

12.0.5

### What operating system are you using?

macOS

### Operating System Version

MacOS Catalina 10.15.7

### What arch are you using?

x64

### Last Known Working Electron version

did not try on preview version

### Expected Behavior

New window should open on clicking a link with target="_blank" in webview irrespective of value of `nativeWindowOpen` when `allowpopups` is added in `<webview>` tag

### Actual Behavior

Application get crashed when I click a link with `target="_blank"` in `<webview>` when I add `allowpopups` in `<webview>` and add `nativeWindowOpen: true` in `webPreferences` in `mainWindow`.

Crashing error is `exited with signal SIGSEGV`

`main.js`
```javascript
const mainWindow = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      preload: path.join(__dirname, 'preload.js'),
      nativeWindowOpen: true,
      webviewTag: true
    }
  })
```
`index.html` is
```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self'">
    <meta http-equiv="X-Content-Security-Policy" content="default-src 'self'; script-src 'self'">
    <title>Hello World!</title>
  </head>
  <body>
    <h1>Hello World!</h1>
    <webview id="foo" src="https://www.w3schools.com/tags/att_a_target.asp" style="display:inline-flex; width:640px; height:480px" allowpopups></webview>

    <h1>click on Try it yuorself button in webview</h1>

    <!-- You can also require other files to run in this process -->
    <!-- <script src="./renderer.js"></script> -->
  </body>
</html>
```

**Comment by BlackHole1** [comment]
This bug was fixed in Electron v13.1.6

see: https://github.com/electron/electron/pull/29874

You can use electron 12.0.14 or higher, or electron 13.1.6 or higher.

They will work normally

**Comment by BlackHole1** [comment]
cc @codebytere you can now close this issue

**Comment by alokmahor** [comment]
> This bug was fixed in Electron v13.1.6
> 
> see: #29874
> 
> You can use electron 12.0.14 or higher, or electron 13.1.6 or higher.
> 
> They will work normally

Thank you
